test/TestRunner.py

```diff
@@ -22,15 +22,15 @@
 import unittest
 
 import ansiblelint
-from ansiblelint import Runner, RulesCollection
+from ansiblelint import default_rulesdir, Runner, RulesCollection
```

Comment:
I'm not sure if using this var makes sense in tests but let's keep it.

Reply:
The default_rules is the same as what the the setup in the class was doing.

Reply:
Yep, I just think that we could be testing a more generic case or many cases w/o depending on this.